lib/lib_rss.php

```diff
@@ -578,3 +578,18 @@ function validateShortcutList($shortcuts) {
 	}
 	return $shortcuts_ok;
 }
+
+function errorMessage($errorTitle, $error = '') {
```

Comment:
As the `$error` variable is optional, it will be better to handle that case to avoid the empty `h2` tag.
I would add something like:
```php
if ('' !== $error) {
	$error = "<h2>{$error}</h2>";
}
```
before the `$message` definition.
Then in the `$message` definition, I would replace `<h2>{$error}<h2>` by `{$error}`.